packages/web/src/ssr/util.ts

```diff
@@ -1,35 +1,8 @@
-import {
-  sdk,
-  DiscoveryNodeSelector,
-  productionConfig,
-  stagingConfig,
-  developmentConfig
-} from '@audius/sdk'
 import { resolveRoute } from 'vike/routing'
 import type { PageContextServer } from 'vike/types'
 
 import { staticRoutes } from 'utils/route'
 
-const sdkConfigs = {
```

Comment:
can we try moving this to a separate file and see if it works? my guess is that ssr/util.ts was being imported in a different context, causing this stuff to be evaluated before we wanted to?

Reply:
Yeah it is getting imported in the +route files

Reply:
updated, good call 👍